recipes/cpptrace/all/conandata.yml

```
@@ -1,5 +1,9 @@
 sources:
   # Newer versions at the top
+  "0.5.3":
+    url:
+      - "https://github.com/jeremy-rifkin/cpptrace/archive/refs/tags/v0.5.3.tar.gz"
```

```suggestion
    url: "https://github.com/jeremy-rifkin/cpptrace/archive/refs/tags/v0.5.3.tar.gz"
```
Maybe it'll make the bot put the bump label ? (but it'll invalidate @RubenRBS 's review, so you probably don't want to touch the file. I'm mentioning it for the next PR (and for bot fixing)